### ISSUE TITLE
Allow left and middle alignment in markdown tables (meta)

### ISSUE THREAD

**Issue by HaleTom** [jupyterlab/jupyterlab#6283]
## Description

Markdown tables all render right aligned.

The following `l` and `m` should be left aligned respectively:
```
| Left | Miiiiiiidle | Right |
|:-----|:-----------:|------:|
| l | m | r |
```
(`jupyter` v4.4.0)

### Table alignment in Markdown cells is ignored. #3180
The last comment was "This may work already in master now that #1812 is fixed."

### Left align tables, honour markdown alignment #2151
Last comment was a "todo" from 18 Jul 2018.

Both issues are closed but the issues described still remain.

## Next actions(?)

* Is there an upstream issue raised? 
* Are the 18 Jul 2018 "todo"s still valid?

**Comment by jasongrout** [comment]
It appears that marked, our markdown parser, [supports it](https://marked.js.org/demo/?text=%7C%20Left%20%7C%20Miiiiiiidle%20%7C%20Right%20%7C%0A%7C%3A-----%7C%3A-----------%3A%7C------%3A%7C%0A%7C%20l%20%7C%20m%20%7C%20r%20%7C&options=%7B%0A%20%22baseUrl%22%3A%20null%2C%0A%20%22breaks%22%3A%20false%2C%0A%20%22gfm%22%3A%20true%2C%0A%20%22headerIds%22%3A%20true%2C%0A%20%22headerPrefix%22%3A%20%22%22%2C%0A%20%22highlight%22%3A%20null%2C%0A%20%22langPrefix%22%3A%20%22language-%22%2C%0A%20%22mangle%22%3A%20true%2C%0A%20%22pedantic%22%3A%20false%2C%0A%20%22sanitize%22%3A%20false%2C%0A%20%22sanitizer%22%3A%20null%2C%0A%20%22silent%22%3A%20false%2C%0A%20%22smartLists%22%3A%20false%2C%0A%20%22smartypants%22%3A%20false%2C%0A%20%22tables%22%3A%20true%2C%0A%20%22xhtml%22%3A%20false%0A%7D&version=master).

 I just tried it in master, and it works: 
<img width="173" alt="Screen Shot 2019-04-30 at 6 50 08 AM" src="https://user-images.githubusercontent.com/192614/56966568-e67bb980-6b4e-11e9-8145-b25318db3482.png">

Perhaps you just need to upgrade to the latest prerelease to see it? We just released 1.0a3.

**Comment by jasongrout** [comment]
Closed as fixed (in master) and will be in the next release. Thanks for double-checking here!
